app/src/main/res/values/strings.xml

```diff
@@ -76,4 +76,7 @@
     <string name="external_player_vlc_player_description">Free and open source cross-platform multimedia player that plays most multimedia files.</string>
     <string name="external_player_system_default">System default</string>
     <string name="external_player_system_default_description">Let the system handle the player choice, some players may not be fully compatible.</string>
+
+    <string name="pref_category_downloads">Downloads</string>
```

Comment:
An extra category for downloads seems a bit unnecessary, I think "General" or something similar would be preferable (as already written above).